### PR TITLE
Add validation geom empty

### PR DESCRIPTION
**Pull request by LePetitTim**

### CODE REVIEW

**Comment by LePetitTim** [comment]
https://github.com/Terralego/django-geostore/issues/49

File: geostore/tests/test_views/test_features.py [hunk 1]
```diff
@@ -238,6 +238,23 @@ def test_feature_put_replace_properties(self):
...
+    def test_feature_put_polygon_empty_wkt(self):
+        response = self.client.put(self.detail_url, data={'geom': 'POLYGON EMPTY',
+                                                          "properties": {"name": "Divona"}})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.json()
+        self.assertListEqual(sorted(list(data['properties'].keys())),
+                             sorted(['name', ]), data)
+
+    def test_feature_put_polygon_empty_geojson(self):
+        response = self.client.put(self.detail_url, data={'geom': '{"type": "Point", "coordinates": []}',
```
Comment (submarcos) [inline geostore/tests/test_views/test_features.py:250]: this is not a polygon

File: geostore/serializers/__init__.py [hunk 2]
```diff
@@ -11,6 +17,41 @@
...
+        if geometry.empty:
+            raise ValidationError(
+                _('Geometry is empty')
+            )
+        if not geometry.valid:
+            raise ValidationError(
+                _('Geometry is not valid')
+            )
+
+    def to_internal_value(self, value):
```
Comment (submarcos) [inline geostore/serializers/__init__.py:31]: don't override to_internal_value, just add custom validators on field

File: geostore/tiles/funcs.py [hunk 3]
```diff
@@ -11,6 +13,13 @@ def select_format(self, compiler, sql, params):
         return sql, params
 
 
+@BaseSpatialField.register_lookup
+class IsEmpty(Transform):
+    lookup_name = 'isempty'
+    function = 'ST_ISEMPTY'
+    output_field = BooleanField()
+
+
```
Comment (submarcos) [inline geostore/tiles/funcs.py]: put it in geostore.lookups.py

**Review by submarcos — CHANGES_REQUESTED** [review]
There is no entry in changelog

**Review by submarcos — APPROVED** [review]

**Pull request merged**